src/components/Heading/Heading.tsx

```diff
@@ -1,25 +0,0 @@
-import React, { Component } from 'react';
-import classnames from 'classnames';
-import { HtmlH, HtmlHProps, hLevels } from '../../reset/HtmlH/HtmlH';
-export { hLevels };
-
-export interface HeadingProps extends HtmlHProps {
-  /** Change HTML-element */
-  variant: hLevels;
-  className?: string;
```

Comment:
Should we have className explicitly mentioned in the core level API as it was also here even if already provided by HtmlHProps?

Reply:
Might be a good idea. Should we also document the 'as' prop since it's used in the examples (Sami's comment from elsewhere)?

Reply:
Looks good!